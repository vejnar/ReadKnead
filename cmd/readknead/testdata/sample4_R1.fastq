@HWI-D00306:1079:HKVXXXXX2:1:1101:2491:1987 1:N:0:ATCACG
GTCGCATTCCGGTAATCATCAGATCGGATGAGCACACGTCTGAACTCCAGTCACAAAAAAAAATCTCGTATGCCGT
+
DDDDDIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
@HWI-D00306:1079:HKVXXXXX2:1:1101:2606:1984 1:N:0:ATCACG
CTAGTAGCTGGTTCCCTCCGAAGTTTCCCTCAGGATAGCTGGCGCTCGCCGATCAAGCAGTTTTATCCGGTAAAGC
+
DDDDDIIIIIIIIIIHIIIIIIIIIIIIIIIIIIIHIIIIIIIIIGIIIIIHIIIIHIIIIIIIIIIIIIIIIIII
@HWI-D00306:1079:HKVXXXXX2:1:1101:3026:1930 1:N:0:ATCACG
NGAGAATAGGTTGAGGCCGTTTCGGCCCCAAGGCCTCTAGTCATAGATCGGAAGAGCACACGTCTGAACTCCAGTC
+
#<<DDHIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIHIIIIIIIIIIHIIIIG
@HWI-D00306:1079:HKVXXXXX2:1:1101:3163:1935 1:N:0:ATCACG
NGCGCCTTAACCCGGCGTTCGGTTCATCCCGCAGCACCAGTTCTGCTTACCAAAAATGGCCCACTAGGCGCGTCGC
+
#<DDDIIIIIIIIIIIIIIIIIIIIIIIIIIIHIIIIIIIHIIIIIIIIIIIIHIIHHIIIIIIIIIIIIIIIIII
