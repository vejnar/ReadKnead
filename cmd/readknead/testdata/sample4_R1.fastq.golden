@HWI-D00306:1079:HKVXXXXX2:1:1101:3026:1930 1:N:0:ATCACG
NGAGAATAGGTTGAGGCCGTTTCGGCCCCAAGGCCTCTAGTCAT
+
#<<DDHIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
